.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,50 @@
+# e2e test for Login with google plugin. 
+
+name: CI for WP Google Embeds plugin
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [automation/playwright]
```

Comment:
@pavanpatil1 Update the branch name to master. Develop branch is not present on repo so I think you can exclude that.